docs/how_to/apphooks.rst

```diff
@@ -308,9 +309,7 @@ will look like this::
 
     class OscarApp(CMSApp):
         name = _("Oscar")
-        urls = [
-            patterns('', *application.urls[0])
```

Comment:
ah ah, probably this file have been moved at some point and I missed the rest of the history